graphics/xfractint/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                xfractint
+version             20.04p14
+categories          graphics
+maintainers         @jflude \
+                    openmaintainer
```

Comment:
Please only use a single line for the maintainers. And add your obfuscated email address.

Reply:
done